catalog/spatial/registry/registry-app/pom.xml

```diff
@@ -75,6 +75,10 @@
             <groupId>org.codice.ddf.registry</groupId>
             <artifactId>registry-report-action-provider</artifactId>
         </dependency>
+        <dependency>
```

Comment:
If you wanted to add it to <depdendencyManagement> looks like the other dependencies are listed here: https://github.com/vinamartin/ddf/blob/8091edea2b703cd64cefd76c97232a7e2d989570/catalog/spatial/registry/pom.xml#L90